crawl-ref/source/webserver/games.d/base.yaml

```diff
@@ -0,0 +1,95 @@
+# Games are stored in order. The order of games you define in these files will
+# be preserved, and the files will be loaded in sorted order.
+games:
+  # id must be unique for each game
+  - id: seeded-web-trunk
+    # The name for this game, displayed on the webtiles HTML interface
+    name: DCSS trunk, custom seed
+    # Executable file to run
+    crawl_binary: ./crawl
+    # Directory all player RC files are stored in
+    rcfile_path: ./rcs/
+    # Directory all player macro files are stored in. Generally the same as
+    # rcfile_path for official webtiles servers.
+    macro_path: ./rcs/
+    # Directory the individual player's morgue files are stored in
+    morgue_path: ./rcs/%n
+    # Directory where ttyrec files for active games are written to
+    inprogress_path: ./rcs/running
+    # Directory where ttyrec files are stored for the individual user
+    ttyrec_path: ./rcs/ttyrecs/%n
+    # Directory where sockets are placed. These sockets are used for
+    # communication between the webtiles server and DCSS executable.
+    socket_path: ./rcs
+    # Static content used by the game (eg spritesheets, game HTML). DCSS builds
+    # this as the web/ directory.
+    client_path: ./webserver/game_data/
```

Comment:
all paths need to document what they are relative to

Reply:
done